### PR TITLE
bump compatability for ArchGDAL to 0.9 and GeoInterface to 1

### PR DESCRIPTION
**Pull request by rafaqz**

### CODE REVIEW

File: test/GADM.jl [hunk 1]
```diff
@@ -39,12 +39,12 @@ end
...
     parent = GADM.get("IND")
     @test parent isa NamedTuple
-    @test GeoInterface.geotype(parent.geom[1]) == :MultiPolygon
+    @test GeoInterface.geomtrait(parent.geom[1]) isa MultiPolygonTrait
     # valid country code, children=true
     parent, children = GADM.get("IND";children=true)
     @test parent isa NamedTuple
     @test children isa NamedTuple
-    @test GeoInterface.geotype(parent.geom[1]) == :MultiPolygon
+    @test GeoInterface.mtrait(parent.geom[1]) isa MultiPolygonTrait
```
Comment (juliohm) [inline test/GADM.jl]:
```suggestion
    @test GeoInterface.geomtrait(parent.geom[1]) isa MultiPolygonTrait
```

**Comment by rafaqz** [comment]
Guess this needs a major version bump for using the new GeoInterface?

**Comment by juliohm** [comment]
@rafaqz please do not merge PRs on GADM.jl without a consent from the maintainers.

I will take care of the version bump and other issues before a new release.

**Comment by rafaqz** [comment]
No problem, apologies for my hastiness!

The reason is we have a cross organisation version switch to GeoInterface 1.0 happening currently, and GADM.jl is blocking registration of the new package Leaflet.jl, which uses GADM for testing.

**Pull request merged**